packages/performance-tests/test/rawInserts.ts

```diff
@@ -18,7 +18,7 @@ const iModelName = "Many PhysicalObjects and Relationships";
 const ELEM_COUNT = 100_000;
```

Comment:
right above this (github won't let me comment on it) is `iModelName`. Don't you want to append the branch name to that?

Reply:
don't have access to branch branchName here so adding it to the reporter entries